rllib/scripts.py

```diff
@@ -102,10 +102,12 @@ def run(example_id: str = typer.Argument(..., help="Example ID to run.")):
     example_file = get_example_file(example_id)
     example_file, temp_file = download_example_file(example_file)
     file_type = example.get("file_type", SupportedFileType.yaml)
+    stop = example.get("stop", "{}")
```

Comment:
Btw, sorry, unrelated to this PR, BUT can we automate the file_type recognition (instead of making it yaml by default, detect type from file extension).

Reply:
done